dev_guide/service_accounts.adoc

```diff
@@ -170,6 +170,15 @@ To allow a secret to be mounted by a service account's pods, run:
 $ oc secrets link --for=mount <serviceaccount-name> <secret-name>
 ----
 
+[NOTE]
```

Comment:
here you follow a section discussing --for=pull and --for=mount; it may be good to be clearer that the NOTE refers to --for=mount (which is also implied if no --for= is specified) only.